docs/release_notes.md

```diff
@@ -10,8 +10,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```

Comment:
Any particular reason for removing this section? It's meant as a staging ground for changelog items that are not yet part of a release (I guess they'd be in develop).